acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/permissioning/NodeSmartContractPermissioningIbftStallAcceptanceTest.java

```diff
@@ -73,12 +67,10 @@ public void restartedIbftClusterShouldNotStall() throws IOException {
     bootnode.setPermissioningConfiguration(permissioningConfiguration);
     nodeA.setPermissioningConfiguration(permissioningConfiguration);
     nodeB.setPermissioningConfiguration(permissioningConfiguration);
-    nodeC.setPermissioningConfiguration(permissioningConfiguration);
-    nodeD.setPermissioningConfiguration(permissioningConfiguration);
 
-    permissionedCluster.start(bootnode, nodeA, nodeB, nodeC, nodeD);
+    permissionedCluster.start(bootnode, nodeA, nodeB);
 
     // Verify blockchain is progressing
-    waitForBlockHeight(bootnode, 15);
+    bootnode.verify(blockchain.reachesHeight(bootnode, 1, 120));
```

Comment:
Think we should check all nodes have progressed not just the bootnode. So something like permissionedCluster.verify(blockchain.reachesHeight(bootnode, 1, 120))

Reply:
done